### ISSUE TITLE
ICO's without a Minimal Valuable Product

### ISSUE THREAD

**Issue by sgarciafer** [Hdactech/Hdac#2]
The blockchain ecosystem has evolved a lot since those times when someone could arrive to an ICO with a simple white paper and claim millions of USD. Therefor, I would suggest you address this issue before Nov 27th. Kind regards!

**Comment by Casslin** [comment]
I would agree with @sgarciafer Please show us some code before we show you the coins!
